hello/chart/validPassthrough.ts

```diff
@@ -0,0 +1,20 @@
+import type { Static } from '@sinclair/typebox'
+import type { ErrorObject } from 'ajv'
+import type { HistoricalDataRequest } from './HistoricalDataRequest.js'
+import type { HistoricalDataResponse } from './HistoricalDataResponse.js'
+import { validator } from './validator.js'
+
+export const validPassthrough = (
```

Comment:
We don't need a separate validPassthrough for both in and outgoing messages. 

Add responses to the existing validPassthrough.